plugins/woocommerce-blocks/assets/js/blocks/product-gallery/frontend.tsx

```diff
@@ -40,6 +40,7 @@ const closeDialog = ( context: ProductGalleryContext ) => {
 	context.isDialogOpen = false;
 	// Reset the main image.
 	context.selectedImage = context.firstMainImageId;
+	document.body.classList.remove( 'modal-open' );
```

Comment:
I wonder if it would be better to namespace this classname so that there is no chance of a conflict if some other extensions are doing the same. Perhaps `wc-block-product-gallery-modal-open`? WDYT?

Reply:
I agree it might be nice to namespace

Reply:
Great suggestion! Included in 56537ac52da08e0e8b37cf0e3353c71a75731dd3